### DISCUSSION TITLE
Any plans to support React 17?

### DISCUSSION THREAD

**Discussion by patwickdane** [clauderic/dnd-kit#614] [Q&A]
Are there any plans to support React 17 anytime soon?

**Comment by Brian-McBride** [comment]
I'm just starting to use this lib, but it seems to be fine on React 17

**Comment by Chevindu** [comment]
I too am keen to migrate my project from [react-sortable-hoc](https://github.com/clauderic/react-sortable-hoc) to dnd-kit and I use React 17. Would help with the warnings if we can get [the peer dependency](https://github.com/clauderic/dnd-kit/blob/master/packages/sortable/package.json#L33) updated. For kind consideration of @clauderic 🙏
